edgepi_rpc_client/client_rpc_channel.py

```diff
@@ -3,7 +3,7 @@
 import logging
 from google.protobuf import service
 import zmq
-from edgepirpc.protos import rpc_pb2 as rpc_pb
+from python_rpc.protobufs import rpc_pb2 as rpc_pb
```

Comment:
pkg name should not be `python_rpc.protobufs`. it should be `edgepi_rpc_protobuf`